### DISCUSSION TITLE
Source code not found on UI since upgrade from 2.5.3 to 2.8.1 in EKS using Helm chart 1.12.0

### DISCUSSION THREAD

**Discussion by dgrahul1991** [apache/airflow#37584] [Q&A]
Apache Airflow version
migrated from 2.5.3 to  Airflow 2.8.1 version

What happened
Airflow version 2.8.1
We get An error occurred while fetching the dag source code. on airflow web user interface, Dag source code not found  in UI 2.8.1
 
 And the dags folder is attached correctly in the web server container.
 
<img width="1722" alt="image" src="https://github.com/apache/airflow/assets/30005759/8b95db63-77e1-42c1-994c-9ff26eb1d692">

**Comment by dgrahul1991** [comment]
Hi @potiuk any Inputs on this will be a great help or if more info needed from my side let me know

**Reply by potiuk** [reply]
No. Generally people here where they have time and feel they can help, Generally calling those people if you do not know if they can or will to spend their time is not advised - because I have no time to look at it, and by calling me you decrease a chance someone else will look at it.

**Reply by dgrahul1991** [reply]
Thanks for reply
sorry for tagging :)

**Reply by potiuk** [reply]
Suggestion 1: We have just released RC1 for 2.8.2 (see our devlist - you will find all the links to devlist in the `community` page of our docs. Try it and see if it is fixed (and of course provide feedback that you tested it in the https://github.com/apache/airflow/issues/37617

Suggestion 2: make sure you have the same secret key: https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#secret-key - configured in all your components - workers, ui, scheduler. This is a common source of problems where this key gets misconfigured. I believe some of the recent fixes made the source code properly not accessible when you have wrong webserver key configured.

**Comment by jabjak** [comment]
is there a solution? I'm facing the same problem after an upgrade. I set a value for AIRFLOW__WEBSERVER__SECRET_KEY
